packages/core/src/util/initializedTooLateHeuristic.js

```diff
@@ -16,15 +16,19 @@ let hasBeenInitializedTooLate = false;
 
 let patterns = [
   /\/@apollo\/gateway\/dist\//,
+  /\/@aws-sdk\/smithy-client\//,
```

Comment:
The only new pattern is `/\/@prisma\/client\//`, the rest of the diff comes from restoring alphabetical order, scnr.